1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,64 @@
-// Write your code here
+let slideshowImages = [
+  "./imgs/1.jpeg",
+  "./imgs/2.jpeg",
+  "./imgs/3.jpeg",
+  "./imgs/4.jpeg",
+  "./imgs/5.jpeg",
+];
+const autoBackButton = document.querySelector("#auto-back");
+const backButton = document.querySelector("#back");
+const stopButton = document.querySelector("#stop");
+const forwardButton = document.querySelector("#forward");
+const autoForwardButton = document.querySelector("#auto-forward");
+const mainImg = document.querySelector("#img");
```

Comment:
Use const instead of let for elements that won't change: For example, the mainImg and commentImg elements are not going to change, so you can use const instead of let